### PR TITLE
420 embellishments

### PR DESCRIPTION
**Pull request by Nikomus**
Trello card: https://trello.com/c/LIkXn5zI/420-embellishments

## Description

- Makes the definition and individual help pages match the rest of the site
- fixes buttons being too close to the bottom of the page

image

@neontribe/sea-map

### CODE REVIEW

File: src/Theme.elm [hunk 1]
```diff
@@ -1,8 +1,9 @@
-module Theme exposing (container, containerContent, globalStyles, green, grey, gridStyle, lightGreen, lightGrey, lightOrange, lightPink, lightPurple, lightTeal, navItemStyles, navLinkStyle, navListStyle, oneColumn, orange, pageHeadingStyle, pink, pureWhite, purple, shadowGrey, teal, threeColumn, twoColumn, verticalSpacing, waveStyle, white)
+module Theme exposing (container, containerContent, expanderButtonStyle, expanderClosedStyle, expanderDefinitionStyles, expanderHeadingStyle, expanderItemStyle, expanderOpenStyle, expanderSymbolStyle, globalStyles, green, grey, gridStyle, lightGreen, lightGrey, lightOrange, lightPink, lightPurple, lightTeal, navItemStyles, navLinkStyle, navListStyle, oneColumn, orange, pageHeadingStyle, pink, pureWhite, purple, quoteStyle, rotate90Style, shadowGrey, teal, threeColumn, twoColumn, verticalSpacing, waveStyle, white)
```
Comment (katjam) [inline src/Theme.elm]: So many!

File: src/Theme.elm [hunk 2]
```diff
@@ -247,6 +249,114 @@ navLinkStyle =
...
+    ]
+
+
+quoteStyle : Style
+quoteStyle =
+    batch
+        [ paddingLeft (rem 1)
+        , before [ property "content" "'\"'" ]
+        , after [ property "content" "'\"'" ]
+        , nthOfType "1n"
```
Comment (katjam) [inline src/Theme.elm:372]: oooh a rainbow!

File: src/View/Definition.elm [hunk 3]
```diff
@@ -52,6 +50,7 @@ view model =
                     ]
                 ]
             ]
+        , verticalSpacing 2
```
Comment (katjam) [inline src/View/Definition.elm:53]: Did we want this in the page tpl in Main?

File: src/View/Definition.elm [hunk 4]
```diff
@@ -77,17 +76,17 @@ renderTerm : Model -> CategoryDefinition -> Html Msg
 renderTerm model category =
     if categoryIsExpanded model category.title then
         dt [ css [ expanderItemStyle ] ]
-            [ button [ onClick (ToggleCategory category.title), css [ expanderButtonStyle ] ]
+            [ button [ onClick (ToggleCategory category.title), css [ expanderButtonStyle, expanderOpenStyle ] ]
                 [ h2 [ css [ expanderHeadingStyle ] ] [ text (t category.title) ]
-                , span [ css [ expanderSymbolStyle, rotate90Style ] ] [ text ">" ]
+                , img [ css [ expanderSymbolStyle, rotate90Style ], src "/sea-map/Arrow.svg" ] []
```
Comment (katjam) [inline src/View/Definition.elm]: Nice

File: src/View/Definition.elm [hunk 5]
```diff
@@ -165,105 +164,17 @@ splitOnEndKeyword partial =
...
 
 
 keywordStyle : Style
 keywordStyle =
     batch
         [ backgroundColor lightGreen
         ]
-
-
-expanderButtonStyle : Style
```
Comment (katjam) [inline src/View/Definition.elm:186]: I guess you've decided these will remain identical? I had small hopes those 2 pages would get different personalities.

File: src/View/HelpSelfGrid.elm [hunk 6]
```diff
@@ -91,6 +92,21 @@ gridItemStyle =
...
         [ flex3 zero zero twoColumn
         , margin (rem 1)
+        , position relative
         , withMedia [ only screen [ Media.minWidth (px 576) ] ]
-            [ flex3 zero zero threeColumn ]
+            [ flex3 zero zero threeColumn
+            , top (rem 3)
+
+            -- nthOfType can take a formula, but as the 7th item is wrapping
+            -- to the centre, it's easier to be explicit
```
Comment (katjam) [inline src/View/HelpSelfGrid.elm:100]: :+1:

**Review by katjam — DISMISSED** [review]
Looks good but consider moving that vertical space to the main page layout function?

File: src/View/Definition.elm (hunk 3 above)
Comment (Nikomus) [reply src/View/Definition.elm:53]: ... that makes sense

File: src/View/Definition.elm (hunk 5 above)
Comment (Nikomus) [reply src/View/Definition.elm:186]: They are the same component, atm so I think easier to look after in one place. It's easy enough to override styles with a more specific one in a page, if we have a need.

File: src/Theme.elm [hunk 7]
```diff
@@ -109,7 +110,7 @@ maxMobile =
 
 withMediaTabletOrDesktop : List Style -> Style
 withMediaTabletOrDesktop =
```
Comment (katjam) [inline src/Theme.elm:112]: This function name should change to `withMediaTablet` ?

**Review by katjam — APPROVED** [review]
Looking good! :jack_o_lantern:

**Pull request merged**